editoast/src/views/projection.rs

```diff
@@ -445,16 +474,18 @@ impl TrainToProjectOnOperationalPoint {
             let first_path_item = refs.next().flatten();
 
             let refs = refs.flatten().map(|(op_ref, id)| {
-                schedule_inputs
+                arrival_inputs
                     .get(&id)
-                    .map(|time| OperationalPointRefAndTime {
-                        time: *time,
+                    .map(|arrival| OperationalPointRefAndTime {
+                        arrival: *arrival,
```

Comment:
nitpick: you might be able to destructure the reference directly. Feel free to resolve in any case.


```suggestion
                    .map(|&arrival| OperationalPointRefAndTime {
                        arrival,
```